src/components/units/icon-label-tag/icon-label-tag.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  title: 'Icon label tag',
+  default: 'attention',
+  context: {
+    previewClass: 'fr-bg--dark',
+    modifier: ['attention'],
+    text: 'Attention',
+    icon: 'attention'
+  },
+  variants: [
+    {
+      name: 'Make',
+      context: {
+        modifier: ['make'],
+        text: 'Make',
+        icon: 'make'
```

Comment:
I can't see this `icon` property used anywhere.
Please continuously clean/refactor as you go, inspecting diffs before every commit to sense check what's being added, and inspecting a comparison of the whole branch/PR before requesting approval. It's much better you take the extra time to afford that care as you craft components than for me to spot, if i do, and then do clean up retrospectively. It may seem a bit of a non-issue in this specific case (and it is!), but it applies to everything as a general principle that's essential in mitigating tech-debt.

Reply:
Ah, thanks for pointing that out @methodog . The 'make' icon is prevalent throughout the card designs and is in the screenshot of the design attached to the Jira ticket so I believed it to be integral. I shall take it out if it's not required.

Reply:
I think you must have misunderstood @nickbergquist - my code comment here is regarding the **"icon"** property itself, the whole name/value pair. I don't see it being referenced in the `html` file. I think it's rendered redundant by the use of the `modifier` property which has the same value, in which case the `icon` name/value pair is not needed.
Your reply refers to the specific value `make` of one of the contexts in this JSON file, and assumes i'm suggesting it's not needed - but if you read my code comment immediately previous to this one, you see I list `make` amongst others which **are** required! However, i do (in that comment above), question the need for an `attention` icon.
Please, try not confuse issues, and take time to read and digest code comments in order.

Reply:
Yes apologies, I did misread your comment and indeed had already identified the 'icon' property for deletion based on the 'modifier' property being used in its place. The 'make' icon-label type is still included and the 'attention' variant is removed, it having been used as a placeholder initially.